src/main/java/seedu/address/logic/commands/AddCommand.java

```diff
@@ -18,30 +17,22 @@
 
```

Comment:
Do you think we should make this more specific? Perhaps call this `AddOpenEndedQuestionCommand` because we would eventually need to have a command to add a multiple-choice question.